test/tests/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+yum install nodejs npm
+npm -g install phantomjs
+
+yum install yum-plugin-copr
+yum copr enable lmr/Autotest
+yum install avocado
+yum install avocado-virt
+
+# because of troubles with cockpit recompiled to /opt/ instead of /usr
+setenforce 0
```

Comment:
Why is cockpit recompiled to /opt? I guess this is just a problem with your machine?